README.md

```diff
@@ -630,21 +630,43 @@ invalid state, which may lead to strange and unexpected behavior.
 
 ## Styles Customization
 
-If you are using SCSS, you can easily customize wizard's global styles and color theme using SCSS variables:
+If you are using SCSS, you can customize wizard's global styles and color theme using SCSS variables:
```

Comment:
you can customize **the** wizard's global styles and color theme...

Reply:
Fixed in 65ba9406a3c02f8bd78029f78fef0a2290f02253